planetiler-core/src/main/java/com/onthegomap/planetiler/expression/Expression.java

```diff
@@ -362,7 +362,7 @@ static MatchAny from(String field, BiFunction<WithTags, String, Object> valueGet
 
       return new MatchAny(field, values,
         Set.copyOf(exactMatches),
-        patterns.isEmpty() ? null : Pattern.compile("(" + Strings.join(patterns, '|') + ")"),
+        patterns.isEmpty() ? null : Pattern.compile("(" + Joiner.on('|').join(patterns) + ")"),
```

Comment:
not sure if I miss anything but there's IMO no need for guava
would personally either do
`Pattern.compile("(" + String.join("|", patterns) + ")")`
or even "better"
`Pattern.compile(patterns.stream().collect(Collectors.joining("|", "(", ")")))`

Reply:
+1, slight preference to using built-in utilities.

Reply:
Thanks for your feedback! I've included the proposed change.